src/web/components/SharingPermission/ParticipantSearchBar.scss

```diff
@@ -1,6 +1,6 @@
 .search-icon {
   color: var(--theme-search-border);
-  height: 1.4375rem;
+  height: 23px;
```

Comment:
This probably should stay as rem - icons are usually pretty much the same as fonts, they should scale with the base font size. If the font size was double but the icon didn't also get bigger, it would probably look wrong.